# Hot fix for LP: #1310846

=== modified file 'etc/maas/templates/power/amt.template'
--- /etc/maas/templates/power/amt.template	2014-02-17 21:00:17 +0000
+++ /etc/maas/templates/power/amt.template	2014-04-14 21:59:46 +0000
@@ -18,10 +18,16 @@
 }
 
 state() {
-  state=`{
-    # Retry the state if it fails because it often fails the first time.
-    amt 2> /dev/null || amt
-  } | grep '^Powerstate:' | awk '{print $2}'`
+  # Retry the state if it fails because it often fails the first time.
+  local count=0
+  local state=
+  while true; do
+    state=$(amt info | grep '^Powerstate:' | awk '{print $2}')
+    [ -n "$state" ] && break
+    count=$((count+1))
+    [ $count -gt 10 ] && break
+    sleep 1
+  done
   echo $state
 }
 
@@ -39,7 +45,7 @@
 echo desired $desired_state
 
 current_state=`state`
-for i in `seq 0 1`; do
+for i in `seq 0 10`; do
   echo found current_state $current_state
   if [ "$current_state" '=' "" ]; then
     echo Cannot get current state

